src/en/news/blog/2021/managing-grafana-dashboards-with-grafonnet/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Managing Grafana Dashboards with Grafonnet"
+date: "2021-09-17"
+author: "Aashish Sharma"
+---
+
+The process of maintaing grafana dashboards or updating the dashboard JSON files can sometimes become a mess as the size of the JSON files becomes too large to debug it. In this blog you can find out how grafana dashboards can be managed as code using Grafonnet.
```

Comment:
Don't be shy, let's talk about actual figures: you know better than anyone how many lines of code and files we need to track. You can also mention the difficulty to test this in CI (and bring up the fact that we are already running automated testing for Prometheus Alertmanager alerts), the fact that this is an area where we receive most of our external contributions (hence it requires close attention), ...

Reply:
Since this is written by someone working on UI stuff, I really miss seeing pictures from the UI, Grafana, how the jsonnet code really looks in Grafana, etc. Don't be afraid of adding pictures.